Resources/Prototypes/Recipes/Lathes/robotics.yml

```diff
@@ -447,6 +447,15 @@
     Glass: 250
     Plastic: 250
 
+- type: latheRecipe
+  id: BorgModuleAdvancedCleaning
+  result: BorgModuleAdvancedCleaning
+  completetime: 3
+  materials:
+    Steel: 250
+    Glass: 250
+    Plastic: 250
```

Comment:
Id personally add the 50 gold like the other advanced modules but it doesnt matter much.

Reply:
Oh yeah, I wasn't paying attention and didn't notice that regular modules no longer need gold. Done